Tests/Scripts/install_dependencies.sh

```diff
@@ -14,12 +14,21 @@ install_from_gh () {
 
 if [[ $1 == "libmongoc" ]]
 then
-	git clone -b ${LIBMONGOC_VERSION} https://github.com/mongodb/mongo-c-driver /tmp/libmongoc
-	pushd /tmp/libmongoc
-	if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr/local; fi
-	if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr; fi
-	sudo make -j8 install
-	popd
+	LIBMONGOC_CACHE_DIR=${HOME}/libmongoc
```

Comment:
Can we add the version to the folder name for posterity?

Reply:
That's a really good idea. We won't need to refresh the cache on libmongoc version bump if the folder name itself is versioned.